src/themes/light.scss

```diff
@@ -20,6 +20,22 @@
 
   --wxc-banner--top--background: #{$wxc-gradation-light-transparent};
   --wxc-banner--bottom--background: #{$wxc-white-alpha-70};
+  --wxc-avatar--placeholder-1--background: #{$wxc-black-alpha-50};
+  --wxc-avatar--placeholder-2--background: #{$wxc-cobalt-70};
+  --wxc-avatar--placeholder-3--background: #{$wxc-cyan-70};
+  --wxc-avatar--placeholder-4--background: #{$wxc-gold-70};
+  --wxc-avatar--placeholder-5--background: #{$wxc-lime-70};
+  --wxc-avatar--placeholder-6--background: #{$wxc-mint-70};
+  --wxc-avatar--placeholder-7--background: #{$wxc-orange-70};
+  --wxc-avatar--placeholder-8--background: #{$wxc-pink-70};
+  --wxc-avatar--placeholder-9--background: #{$wxc-slate-70};
+  --wxc-avatar--placeholder-10--background: #{$wxc-violet-70};
+  --wxc-avatar--placeholder-11--background: #{$wxc-purple-70};
+
+  --wxc-avatar--text-color: #{$wxc-white-alpha-100};
+  
+  --wxc-banner-background: #{$wxc-gradation-light-transparent};
```

Comment:
merge conflict?

also, please put --wxc-avatar* variables in a single block, separated from top/bottom with a single line